.changeset/empty-nails-push.md

```diff
@@ -0,0 +1,24 @@
+---
+'@openfn/language-common': major
+---
+
+### Key Changes
+
+**`logResponse` function:**
```

Comment:
We don't need to document this publicly, or count it as a major change, because it doesn't affect the public facing API. It only affects our internal APIs.

I would probably just call this a very minor patch dependency with a line like `Updated internal logResponse API`. 99% of users don't need to know about this.

Reply:
Makes sense